apps/frontend/src/modules/auth/auth-api.ts

```diff
@@ -1,11 +1,12 @@
+import { type AuthResponseDto } from "@meetlytic/shared";
```

Comment:
The same for other files
```suggestion
import { type AuthResponseDto } from "./libs/types/types.js";
```